docs/source/index.rst

```diff
@@ -1,5 +1,6 @@
-ALNS
-====
+.. figure:: assets/images/logo.svg
+   :alt: ALNS logo
+
```

Comment:
Should we add a `figwidth` here just to be sure?

Reply:
Sure! Added it.